include/expire.php

```diff
@@ -9,14 +9,32 @@ function expire_run(&$argv, &$argc){
 	require_once('include/items.php');
 	require_once('include/Contact.php');
 
+	load_hooks();
+
+	if (($argc == 2) && (intval($argv[1]) > 0)) {
```

Comment:
Good question. I mostly never question my own decisions, they are mostly done intuitively. :-) At the places where I use ```(int)``` I know for sure that the value will be a number. This here is some kind of sanity check. I'm not sure what ```(int)``` is doing with values that for sure aren't integer values.